migration/v32.go

```diff
@@ -0,0 +1,39 @@
+package migration
+
+import (
+	"time"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"github.com/gofrs/uuid"
+	"gorm.io/gorm"
+)
+
+func v32() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "32",
+		Migrate: func(db *gorm.DB) error {
+			if err := db.AutoMigrate(&v32User{}); err != nil {
+				return err
+			}
+			return db.Exec("ALTER TABLE `users` ALTER COLUMN `display_name` VARCHAR(32)").Error
```

Comment:
[should] AutoMigrateは

> AutoMigrate will create tables, missing foreign keys, constraints, columns and indexes. It will change existing column’s type if its **size**, precision, nullable changed.
https://gorm.io/docs/migration.html

とあるので、VARCHARのサイズが変わったら自動で変えてくれそうな気もしますが、変えてくれなかったですか？

Reply:
確かめずに書いたので、あとでやってみます

Reply:
いろいろ試してみた結果、migrationを書かなくてもmodelの構造体についてる長さを変えるだけで変わるっぽいです。v32.goは33文字以上のを丸める処理だけ書きます。
https://github.com/traPtitech/traQ/blob/ea7aecbb0f0027714f4e079e1b9f786f1fd648d1/model/users.go#L93